plugins/node/opentelemetry-instrumentation-graphql/src/enum.ts

```diff
@@ -53,3 +53,20 @@ export enum SpanNames {
   SCHEMA_VALIDATE = 'graphql.validateSchema',
   SCHEMA_PARSE = 'graphql.parseSchema',
 }
+
+export const createResolveSpanName = (fieldName: string): string => {
+  return `${SpanNames.RESOLVE}.${fieldName}`;
+};
+
+export const createExecuteSpanName = (
+  operationType?: string,
+  operationName?: string
```

Comment:
Is name infinite cardinality or limited? The spec asks span names to have relatively low cardinality

Reply:
It's infinite, as clients can set anything as operation name.[1] I have removed it from execute span name.

[1] https://graphql.org/learn/queries/#operation-name